src/ostorlab/runtimes/local/models/models.py

```diff
@@ -483,3 +493,206 @@ def is_valid(api_key: str) -> bool:
+    application_name = sqlalchemy.Column(sqlalchemy.String(255))
+
+    __mapper_args__ = {
+        "polymorphic_identity": "android_store",
+    }
+
+    @staticmethod
+    def create(package_name: str = "", application_name: str = ""):
+        """Persist the android store information in the database.
+
+        Args:
+            package_name: The application identifier.
+            application_name: The application name as shown in the store.
+
+        Returns:
+            android store object.
+        """
+        with Database() as session:
+            asset = AndroidStore(
+                package_name=package_name,
+                application_name=application_name,
+            )
+            session.add(asset)
+            session.commit()
+            return asset
+
+
+class IosFile(Asset):
+    __tablename__ = "ios_file"
+    id = sqlalchemy.Column(
+        sqlalchemy.Integer, sqlalchemy.ForeignKey("asset.id"), primary_key=True
+    )
+    bundle_id = sqlalchemy.Column(sqlalchemy.String(255))
+    path = sqlalchemy.Column(sqlalchemy.String(1024))
+
+    __mapper_args__ = {
+        "polymorphic_identity": "ios_file",
+    }
+
+    @staticmethod
+    def create(bundle_id: str = "", path: str = ""):
+        """Persist the iOS file information in the database.
+
+        Args:
+            bundle_id: The application identifier.
+            path: Local/Remote path of the IPA.
+
+        Returns:
+            iOS file object.
+        """
+        with Database() as session:
+            asset = IosFile(
+                bundle_id=bundle_id,
+                path=path,
+            )
+            session.add(asset)
+            session.commit()
+            return asset
+
+
+class IosStore(Asset):
+    __tablename__ = "ios_store"
+    id = sqlalchemy.Column(
+        sqlalchemy.Integer, sqlalchemy.ForeignKey("asset.id"), primary_key=True
+    )
+    bundle_id = sqlalchemy.Column(sqlalchemy.String(255))
+    application_name = sqlalchemy.Column(sqlalchemy.String(255))
+
+    __mapper_args__ = {
+        "polymorphic_identity": "ios_store",
+    }
+
+    @staticmethod
+    def create(bundle_id: str = "", application_name: str = ""):
+        """Persist the iOS store information in the database.
+
+        Args:
+            bundle_id: The application identifier.
+            application_name: The application name as shown in the store.
+
+        Returns:
+            iOS store object.
+        """
+        with Database() as session:
+            asset = IosStore(
+                bundle_id=bundle_id,
+                application_name=application_name,
+            )
+            session.add(asset)
+            session.commit()
+            return asset
+
+
+class Url(Asset):
+    __tablename__ = "urls"
+    id = sqlalchemy.Column(
+        sqlalchemy.Integer, sqlalchemy.ForeignKey("asset.id"), primary_key=True
+    )
+    links = sqlalchemy.Column(sqlalchemy.String(1024))
```

Comment:
why string ? i know ARRAY is not supported in sqlite but you can use json i think its better